hack/setup-local-multi-cluster.sh

```diff
@@ -5,29 +5,31 @@
 export DOCKER_HOSTNAME=localhost:5000
 export DOCKER_NAMESPACE=m4d-system
 export HELM_EXPERIMENTAL_OCI=1
+export VALUES_FILE=m4d/kind-control.values.yaml
 
 make kind-setup-multi
 kubectl config use-context kind-control
 make -C third_party/razee all
 
 kubectl config use-context kind-control
-make cluster-prepare
+make -C charts vault
+make -C charts cert-manager
+kubectl apply -f https://raw.githubusercontent.com/IBM/dataset-lifecycle-framework/master/release-tools/manifests/dlf.yaml
 make docker-minimal-it
 make cluster-prepare-wait
 make -C secret-provider configure-vault
-make -C secret-provider deploy
-make -C manager deploy-crd
-make -C manager deploy_multi_it
+make -C charts m4d
 make -C manager wait_for_manager
 make -C modules helm-chart-push
 
+export VALUES_FILE=m4d/kind-kind.values.yaml
 kubectl config use-context kind-kind
-make cluster-prepare
+make -C charts vault
+make -C charts cert-manager
+kubectl apply -f https://raw.githubusercontent.com/IBM/dataset-lifecycle-framework/master/release-tools/manifests/dlf.yaml
 make cluster-prepare-wait
```

Comment:
I think here too only waiting for dlf is needed.

Reply:
Yes. But aggregating the waiting had a point of being able to deploy many things in parallel and then have one huge wait loop before continuing. This sped up the tests and deployments quite a bit. An alternative is to call all these targets in parallel.